### PR TITLE
Improve flexbox batch block

### PR DESCRIPTION
**Pull request by gsambrotta**
PR for the Issue #399

### CODE REVIEW

**Review by lukestanley — CHANGES_REQUESTED** [review]
This looks good.
Why hide overflow? Is that better than allowing scrolling?
I think it would be good to add to the documentation more about the block and maybe mention other blocks it's good with?
A bit more detail about it's current usage that's evolved might help.

**Comment by gsambrotta** [comment]
> Why hide overflow? Is that better than allowing scrolling?
I hide the overflowing content of the whole container, so not specific cards. It doesn't actually overflow, but is a precaution to be sure that all the cards will fit in the container.
In this situation, I will avoid scrolling. The cards have to resize or fall in another row. 
I would say we never want the situation in which a user has to scroll to see more cards. Why? It is ugly and not user-friendly at all.

**Comment by lukestanley** [comment]
If the `overflow: hidden` property it's just a precaution, for a situation that we've not ran into, isn't it best to not have it there and keep the CSS more minimal? @gsambrotta

File: src/app/blocks/batch-block/batch-block.component.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+:host(.batch-block-flex) {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 10px 20px;
+    overflow: hidden;
```
Comment (lukestanley) [inline src/app/blocks/batch-block/batch-block.component.scss:7]: If the `overflow: hidden` property it's just a precaution, for a situation that we've not ran into, isn't it best to not have it there and keep the CSS more minimal? @gsambrotta

File: src/app/blocks/batch-block/batch-block.component.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+.multi-wrapper {
+  flex: 1;
+  overflow: hidden;
```
Comment (lukestanley) [inline src/app/blocks/batch-block/batch-block.component.scss:12]: Same thing here, is this needed? @gsambrotta

File: src/app/blocks/batch-block/batch-block.component.scss (hunk 1 above)
Comment (gsambrotta) [reply src/app/blocks/batch-block/batch-block.component.scss:7]: Yeah, I was also battling with myself to decide whether I should leave it or not. 
Let's remove it. It should not make any difference.

**Review by lukestanley — APPROVED** [review]
Lovely stuff.

**Pull request merged**